pkg/domain/entities/images.go

```diff
@@ -309,30 +309,30 @@ type ImageSaveOptions struct {
 	Quiet bool
 }
 
-// ImageScpOptions provide options for securely copying images to podman remote
+// ImageScpOptions provide options for securely copying images to and from a remote host
 type ImageScpOptions struct {
-	// SoureImageName is the image the user is providing to load on a remote machine
-	SourceImageName string
-	// Tag allows for a new image to be created under the given name
-	Tag string
-	// ToRemote specifies that we are loading to the remote host
-	ToRemote bool
-	// FromRemote specifies that we are loading from the remote host
-	FromRemote bool
+	// Remote determines if this entity is operating on a remote machine
+	Remote bool `json:"remote,omitempty"`
+	// File is the input/output file for the save and load Operation
+	File string `json:"file,omitempty"`
+	// Quiet Determines if the save and load operation will be done quietly
+	Quiet bool `json:"quiet,omitempty"`
+	// Image is the image the user is providing to save and load
+	Image string `json:"image,omitempty"`
+	// Transfer determines whether we are transferring an image locally
```

Comment:
I don't see why this field should exists.  This can be determined from the source and dest 

transfer := !(source.Remote || dest.Remote)

Reply:
I see what you're saying, do you think `podman image scp alpine` should default to transferring to root? currently `podman image scp alpine` is basically a noop and transfer only is triggered via the inclusion of `@localhost`

Reply:
No source and destination should always be required.  No defaults.
podman image scp root@Localhost::alpine .
podman image scp alpine root@Localhost::

Reply:
okay so then I think the transfer bool is necessary because `!source.remote || !dest.remote is the equivalent to podman image scp alpine`